tools/bridge/bridge/main.py

```diff
@@ -241,6 +238,21 @@ def reload_logging_config(config_path):
         )
 
 
+def install_signal_handler(signum, name, f, *args, **kwargs):
+    def handler():
+        gevent.getcurrent().name = name
+        logger.info(f"Received {signal.Signals(signum).name} signal.")
+        f(*args, **kwargs)
+
+    gevent.signal(signum, handler)
+
+
+def log_internal_state(recorder):
+    while True:
+        gevent.sleep(60.0)
```

Comment:
Probably nice to have this configurable or at least a constant in `constants.py`..

Reply:
I guess I will make this configurable then, maybe, sometime in the future.